test/token/ERC20/ERC20.behavior.js

```diff
@@ -178,6 +178,17 @@ function shouldBehaveLikeERC20 (errorPrefix, initialSupply, initialHolder, recip
         );
       });
     });
+
+    describe('when the token owner and the recipient are the zero address', function () {
```

Comment:
This is checking that the `transferFrom` function is not correctly preventing the 0 address.

We need a test of the internal function (`$_update`) to check that doing from 0 to 0 internally does not revert, emits the right event, and doesn't change the totalsupply